roles/sft-server/tasks/configure.yml

```diff
@@ -137,7 +147,7 @@
 - name: 'generating DH parameters for SFT nginx vhost (NOTE: This is going to take a long time)'
   openssl_dhparam:
     path: "{{ nginx_dhparams_path }}"
-    size: 4096
+    size: 1024
```

Comment:
This change is not reflected in the PR description - what's the motivation to reduce key size?

Reply:
Oh boy, this on slipped through. I'm sorry. It was just me temp. reducing the deploy time for my test run. Thank you for spotting it!